keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -104,57 +115,35 @@ As a contributor, I want to be able to submit issues or PRs and:
 - have some assurance that they will be routed to the correct group
 - have them addressed in a timely manner
 
-<<[UNRESOLVED]>>
-
-### Notes/Constraints/Caveats (optional)
-
-<!--
-What are the caveats to the proposal?
-What are some important details that didn't come across above.
-Go in to as much detail as necessary here.
-This might be a good place to talk about core concepts and how they releate.
--->
-
-<<[/UNRESOLVED]>>
-
-<<[UNRESOLVED]>>
 
 ### Risks and Mitigations
 
-<!--
-What are the risks of this proposal and how do we mitigate.  Think broadly.
-For example, consider both security and how this will impact the larger
-kubernetes ecosystem.
-
-How will security be reviewed and by whom?
 
-How will UX be reviewed and by whom?
+**The new labels and process are either ignored or fall into dis-use.**
 
-Consider including folks that also work outside the SIG or subproject.
--->
+There are close to 200 labels associated with the [kubernetes/kubernetes]
+repository. While this removes some of the labels it does introduce new ones and
+adds an additional process. Both of these could potentially go unused or ignored
+without effort made by the community groups to use them appropriately.
 
-<<[/UNRESOLVED]>>
+When the new process is ready to be put into place, the [upstream marketing team]
+will be engaged to ensure there is clear communication regarding the changes.
+Their message will be backed by updated documentation regarding the new
+[issue triage process].
 
-## Design Details
 
-<!--
-This section should contain enough information that the specifics of your
-change are understandable.  This may include API specs (though not always
-required) or even code snippets.  If there's any ambiguity about HOW your
-proposal will be implemented, this is the place to discuss them.
--->
+**Removing the least used labels disrupts some contributors workflow**
 
-### Graduation Criteria
+While `triage/duplicate`, `triage/not-reproducible`, and `triage/unresolved` are
```

Comment:
/cc @BenTheElder -- wanted to check if you still had concerns around removing these labels for k/k (https://github.com/kubernetes/test-infra/pull/16298#issuecomment-648482752)?

Reply:
I consider duplicate and not-reproducible very useful labels in all repos.

If I go to close a duplicate issue, I label it as such, so it's known to not be the canonical issue.

If I'm looking through issue triage, I want to know that an issue is currently not-reproducible or needs-information before I waste time looking into it.

unresolved I don't really have a comment on